### ISSUE TITLE
deploy with apache 

### ISSUE THREAD

**Issue by BitForceStudio** [jacobian/channels-example#15]
I got error messages by using apache to handle the http and ws request. 
It runs with manage.py runserver.
If use apache, it will return error code 200. 
Anyone did try this before?

**Comment by jasjuang** [comment]
@BitForceStudio I ran into the same problem. Did you managed to solve it?

**Comment by BitForceStudio** [comment]
Yes, but I forgot how I solve it in detail. I think what I did is change the server from apache to daphne and then use runwork.  the apache didn't support channel yet. You can find some useful work in here: http://stackoverflow.com/questions/37098504/how-to-configure-apache-to-run-asgi-in-django-channels-is-apache-even-required

or the twist version problem, have to be 16.2.0 as I remembered.  Try this first

**Comment by jasjuang** [comment]
Thanks a lot for the pointers. 

My twisted version is exactly 16.2.0. If your link is not outdated yet it explains why it's not working... I am so surprised Apache doesn't support ASGI. The problem for me is I am using Elastic Beanstalk to deploy my application and Apache comes with default and there is no way for me to choose other web servers like daphne or nginx ...
